packages/domain-picker/src/info-tooltip/style.scss

```diff
@@ -0,0 +1,12 @@
+.info-tooltip {
+	.components-popover__content > div {
+        color: var( --color-neutral-50 );
+        font-size: $font-body-small;
+        padding: 16px;
+        text-align: left;
+
+        a {
+            color: var( --studio-blue-40 );
+        }
+    }
+}
```

Comment:
Make sure to have `stylelint` configured. Missing empty line at end made me think it might not be the case. :-) (Tho I think CI should run it, but maybe not for packages).

Reply:
You're right that my stylelint isn't configured correctly (going to fix this also) although this does pass linting rules I believe. I can't see any empty line rule in the `.stylelintrc` and also running `npm run lint:css` doesn't flag any linting errors for this file.

Reply:
You're right! It's actually my Editor doing it because of this config:

https://github.com/Automattic/wp-calypso/blob/f9632e9eeeca5427f14d497c00a46c3fad9f04d2/.editorconfig#L15